docs/specification.md

```diff
@@ -11,7 +11,7 @@ is intended to provide a baseline for commonality between implementations.
 
 ## 1 - Schema Version
 
-This document describes version 1.0.1 of the schema.
+This document describes version 1.0.2 of the schema.
```

Comment:
What should be the right schema version?

Reply:
I think 1.1 ?